vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
Alright, it looks failing somewhere inside of `shutil.copytree()`'s system calls, which are file permission/type checks. I think we can leave ubuntu here.